Pipfile

```diff
@@ -32,5 +32,6 @@ pyjwt = "==2.5.0"
 pytz = ">=2022.5"
 django = "4.1.x"
 uvicorn = "0.19.x"
+django-oidc-provider = {git = "https://github.com/timonegk/django-oidc-provider-django4"}
```

Comment:
Könntest du das package noch in die Law&Orga organisation moven?
name vllt lawandorga-django-oidc-utils mh weiß nicht vllt fällt dir noch was ein